src/AppRoutes.jsx

```diff
@@ -43,7 +45,23 @@ const AppRoutes = () => {
           <Route index element={<Contacts />} />
           <Route path=":webId" element={<Profile />} />
         </Route>
-        <Route path="/messages" element={<Messages />} />
+        <Route path="/messages">
+          {MESSAGE_PAGES_LIST.map((messagePagesProps) => (
```

Comment:
nit: would this read nicer as `messagePage`? Then we would get `messagePage.path` and `messagePage.element`

Reply:
Agreed, definitely a nicer read. I'll have this switched up.